tools/goctl/util/stringx/string.go

```diff
@@ -49,12 +52,12 @@ func (s String) Source() string {
 	return s.source
 }
 
-// Title calls the strings.Title
+// Title calls the cases.Title
 func (s String) Title() string {
 	if s.IsEmptyOrSpace() {
 		return s.source
 	}
-	return strings.Title(s.source)
+	return cases.Title(language.Dutch).String(s.source)
```

Comment:
Why Dutch?

Reply:
sorry, It is my negligence, I think it's used in mongo generate type, so,  language.English fit well